src/AppBundle/DataFixtures/ORM/LoadPostFixtures.php

```diff
@@ -31,8 +32,9 @@
  *
  * @author Ryan Weaver <weaverryan@gmail.com>
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
  */
-class LoadFixtures extends AbstractFixture implements ContainerAwareInterface
+class LoadPostFixtures extends AbstractFixture implements DependentFixtureInterface, ContainerAwareInterface
```

Comment:
IMO, you can remove the ``Load`` part of the name, now that there is an entity name